packages/cosmic-swingset/lib/anylogger-agoric.js

```diff
@@ -27,8 +27,16 @@ anylogger.ext = (l, o) => {
       l[level] = () => {};
     } else {
       // Enable the printing with a prefix.
-      const doLog = l[level] || (() => {});
-      l[level] = (...args) => doLog(`${prefix}:`, ...args);
+      const doLog = l[level];
+      if (doLog) {
+        l[level] = (...args) => {
+          // Add a timestamp.
+          const now = new Date().toISOString();
```

Comment:
ambient access to the clock... hm. I guess we don't close over this and export it as a function. I guess this whole modules is imported for its effects.

Oh well... logging is often messy. I guess we can deal.

But... if I ever have to touch this file, I reserve the right to re-write it respecting ocap discipline.